src/main/kotlin/nmcp/NmcpPublishTask.kt

```diff
@@ -31,8 +33,13 @@ abstract class NmcpPublishTask : DefaultTask() {
     abstract val password: Property<String>
 
     @get:Input
+    @get:Optional
     abstract val verifyStatus: Property<Boolean>
 
+    @get:Input
+    @get:Optional
+    abstract val verificationTimeout: Property<Int>
```

Comment:
Wdyt of java.time.Duration for the timeout?

Reply:
Works for me 👍 . Was a bit unsure whether it was `Serializable` but it is!